src/QuadForm/LineOrbits.jl

```diff
@@ -29,7 +29,7 @@ function LineEnumCtx(K::T, n::Int) where {T <: Union{fpField, FpField}}
   depth = n + 1
   dim = n
   q = order(K)
-  length = divexact(q^n - 1, q - 1)
+  length = divexact(BigInt(q)^n - 1, q - 1)
```

Comment:
Here

Reply:
This I cannot remove, otherwise all functions like `collect`, or `Vector{Int}(undef, length(P))` will not work if we set the length to be a `ZZRingElem`